apps/explorer-e2e/src/integration/node-switcher.cy.js

```diff
@@ -0,0 +1,82 @@
+const nodeErrorType = 'node-error-type';
+const nodeErrorMsg = 'node-error-message';
+const nodeId = 'node-url-0';
+const customNodeBtn = 'custom-node';
+
+context('Node switcher', function () {
+  before('visit home page', function () {
+    cy.intercept('GET', 'https://static.vega.xyz/assets/capsule-network.json', {
+      hosts: ['http://localhost:3028/query'],
+    }).as('nodeData');
+    cy.visit('/');
+    cy.wait('@nodeData');
+  });
+
+  describe('form validations and responses', function () {
+    this.beforeEach('open node selector', function () {
```

Comment:
question (blocking): Why use `this` keyword here?

Reply:
That's a mistake. Removed `this`